dali/test/python/test_operator_python_function.py

```diff
@@ -479,6 +487,8 @@ def test_sink():
 
 
 counter = 0
+
+
```

Comment:
again, the counter is tightly coupled to the function below.

Reply:
Removed the newlines